### PR TITLE
Feat: Popularity and Relevance sorting on search support

### PR DESCRIPTION
**Pull request by SWH-Relewise**

### CODE REVIEW

File: src/components/SearchOverlay.vue [hunk 1]
```diff
@@ -200,6 +206,12 @@ function searchFor(term: string) {
                                 Showing results for <strong>{{ filters.term }}</strong>
                             </h2> 
                             <span v-if="result.hits > 0">Showing {{ page * 30 - 29 }} - {{ result?.hits < 30 ? result?.hits : page * 30 }} of {{ result?.hits }}</span>
+                            <div class="hidden lg:block lg:flex-grow">
+                            </div>
+                            <select v-model="filters.sort" class="text-sm lg:text-base w-full lg:w-1/6" @change="search">
+                                <option>Relevance</option>
+                                <option>Popular</option>
```
Comment (mzanoni) [inline src/components/SearchOverlay.vue:213]: Please use same sorting options as on `Category`

**Review by mzanoni — CHANGES_REQUESTED** [review]

File: src/components/SearchOverlay.vue (hunk 1 above)
Comment (SWH-Relewise) [reply src/components/SearchOverlay.vue:213]: The price sorting? 
Currently these two options are now the same.
The category page has sorting on prices asc and decs as well.
That makes no sense to do here.

File: src/components/SearchOverlay.vue [hunk 2]
```diff
@@ -66,6 +66,7 @@ function showOrHide(show: boolean) {
 }
 
 function typeAHeadSearch() {
+    console.log(filters.value);
```
Comment (mzanoni) [inline src/components/SearchOverlay.vue:69]: remove :)

File: src/components/SearchOverlay.vue (hunk 1 above)
Comment (mzanoni) [reply src/components/SearchOverlay.vue:213]: Why does that make no sense here?

And yes, I want the same 4 options both places :)
(Could extract helper functions and components to make keeping them in sync easier)

**Review by mzanoni — CHANGES_REQUESTED** [review]

File: src/components/SearchOverlay.vue (hunk 1 above)
Comment (SWH-Relewise) [reply src/components/SearchOverlay.vue]: I have extracted it to it's own component! 😊

**Review by mzanoni — APPROVED** [review]

**Pull request merged**